.storybook/main.js

```diff
@@ -20,4 +20,12 @@ module.exports = {
     autodocs: true,
   },
   staticDirs: ['../static'],
+  viteFinal: async (config) => {
+    console.log('config');
+
+    config.resolve.alias['src/api/services/PortalService'] = require.resolve(
```

Comment:
Is this being used to mock in storybook?

Reply:
yes but i didn't manage to fix it, so forgot about it.... Maybe i can remove this and work on the story after ?

Reply:
Yeah just remove it for now 😄